pandas/_libs/tslibs/tzconversion.pyx

```diff
@@ -167,7 +169,8 @@ timedelta-like}
         # where result_a != result_b and neither of them are NAT)
         both_nat = np.logical_and(result_a != NPY_NAT, result_b != NPY_NAT)
         both_eq = result_a == result_b
-        trans_idx = np.squeeze(np.nonzero(np.logical_and(both_nat, ~both_eq)))
+        trans_idx = np.squeeze(np.nonzero(np.logical_and(both_nat,
+                                                        np.logical_not(both_eq))))
```

Comment:
I don’t think the numpy functions work on memoryviews

Reply:
2nd time that tests pass locally and fail on CI. I'm working on other cython changes and can see the effects after recompilation. It's a mystery to me. push another commit.

Reply:
> I don’t think the numpy functions work on memoryviews

I found the problem, `npy_bool` memoryviews are not supported in Cython 0.29.
This is fixed in the cython's master branch, which I had installed.